main.cpp

```diff
@@ -10,158 +9,249 @@ video: Chapter 2 - Part 3
+like you did in b) 6) if your function returns something other than void, add
+'return { };' at the end of it. 7) provide default values for an arbitrary
+number of parameters in the function parameter list.
+
+8) consult the coding style guide found in the Readme.MD and adjust the
+formatting of your 10 functions.  At this point, you might have something that
+looks like this: float someFunc2(bool yes, int bar=2) { ignoreUnused(yes, bar );
+return { }; } This does not conform with the coding standard for this course
+(check the Readme.MD) and needs to be corrected
+
+9) in the main function at the end:
+    for each of those functions declared,
+        a) write out how the function would look if called with correct
+arguments b) if the function returned anything, store it in a local variable via
+the 'auto' keyword. c) pass the local variables to ignoreUnused() as you did in
+2b) see main() for an example of this.
+
 10) click the [run] button.  Clear up any errors or warnings as best you can.
  */
 
-//2)
-void variableDeclarations()
-{
-    //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+// 2)
+void variableDeclarations() {
+    // example:
+    int number =
+      2; // declaration of a variable named "number", that uses the primitive
+         // type 'int', and the variable's initial value is '2'
+    int height = 70;
+    int weight = 180;
+    
+    float temperature = 70.5f;
+    float speed = 5.5f;
+    float pressure = 14.7f;
+    
+    bool isSad = false;
+    bool isHappy = true;
+    bool isAlive = true;
+    
+    double distance = 5.5;
+    double time = 2.5;
+    double velocity = -5.5;
+    
+    char letter = 'a';
+    char symbol = '$';
+    char digit = '1';
+    
+    unsigned int age = 18;
+    unsigned int population = 4000000;
+    unsigned int count = 10;
+    
+    ignoreUnused(
+      number, height, weight, temperature, speed, pressure, isSad, isHappy,
+      isAlive, distance, time, velocity, letter, symbol, digit, age, population,
+      count); // passing each variable declared to the ignoreUnused() function
 }
 
 /*
  10 functions
  example:
  note: this example shows the result after completing steps 3-8
  */
-bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
-{ 
-    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
-    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
-} 
-
+bool rentACar(
+    int rentalDuration,
+    int carType = 0) // function declaration with random number of arguments,
+                     // arbitrary number of arguments have default value

```

Comment:
The formatting of these function parameters did not need to be changed
- [ ] revise/revert